boards/BMS/Src/Io/Io_CellVoltages.c

```diff
@@ -0,0 +1,145 @@
+#include "Io_CellVoltages.h"
+#include "Io_LTC6813.h"
+#include "Io_SharedSpi.h"
+#include "configs/App_CellConfigs.h"
+#include "configs/Io_LTC6813Configs.h"
+
+#define NUM_OF_CELLS_PER_LTC6813_REGISTER_GROUP 3U
+
+static const uint16_t cell_voltage_register_group_commands
+    [NUM_OF_CELL_VOLTAGE_REGISTER_GROUPS] = {
+        0x0400, // RDCVA
+        0x0600, // RDCVB
+        0x0800, // RDCVC
+        0x0A00, // RDCVD
+        0x0900, // RDCVE
+        0x0B00  // RDCVF
+    };
+
+static uint16_t cell_voltages[NUM_OF_CELL_MONITOR_ICS]
+                             [NUM_OF_CELLS_READ_PER_IC];
+
+/**
+ * Parse cell voltages received from the cell monitoring IC and perform PEC15
+ * checks.
+ * @param current_ic The current cell monitoring IC to parse cell voltages for.
+ * @param current_register_group The current register group on the given IC to
+ * parse cell voltages for.
+ * @param rx_cell_voltages The buffer containing the cell voltages read from the
+ * cell monitoring IC.
+ * @return EXIT_CODE_OK if the PEC15 check was successful. Else,
+ * EXIT_CODE_ERROR.
+ */
+static ExitCode Io_LTC6813_ParseCellsAndPerformPec15Check(
+    size_t  current_ic,
+    size_t  current_register_group,
+    uint8_t rx_cell_voltages[]);
+
+static ExitCode Io_LTC6813_ParseCellsAndPerformPec15Check(
+    size_t  current_ic,
+    size_t  current_register_group,
```

Comment:
Should this be a  `CellVoltageRegisterGroup` instead of just a `size_t`?

Reply:
Nice catch thanks. This is fixed